### ISSUE TITLE
Converting GIF to MP4 to improve performance 

### ISSUE THREAD

**Issue by Rafi993** [thepracticaldev/dev.to#9263]
I was able to insert GIF into the cover image and it was causing lighthouse score to reduce. GIF in the cover should probably converted to MP4 to increase performance and to improve the lighthouse score. Since MP4 files are smaller than GIF

**To Reproduce**

1. Go to dev.to article with GIF as cover image and run the lighthouse
2. Go to dev.to article without GIF as cover image and run the lighthouse

**Expected behavior**
GIF should be converted to mp4 and it should auto play in loops (like [twitter](https://mashable.com/2014/06/20/twitter-gifs-mp4/) does) so that user cannot notice that GIF that is actually not a GIF but MP4.
 
**Screenshots**
### score of page with GIF cover image
![score_with_gif](https://user-images.githubusercontent.com/8092120/87222662-37ae4c80-c393-11ea-9d70-120a2623af67.png)
### score of page without GIF cover image
![score_without_gif](https://user-images.githubusercontent.com/8092120/87222666-3d0b9700-c393-11ea-9883-eee1d051e93d.png)

**Desktop:**

- OS: elementary OS 5.1.6 Hera  
- Browser: Chrome
- Version: 83.0.4103.116 (Official Build) (64-bit)

**Additional context**

https://web.dev/replace-gifs-with-videos/
https://web.dev/efficient-animated-content/

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Thanks @Rafi993, it's really interesting! We would also solve a few issues with GIF images other than size.

**Comment by nickytonline** [comment]
Closing as this is a duplicate of #3638 .

**Comment by rhymes** [comment]
@Rafi993 could you kindly copy your comment over to #3638 ?
